src/bci_build/package/stunnel/README.md.j2

```diff
@@ -0,0 +1,71 @@
+# {{ image.pretty_name }} Container Image
+
+{% include 'badges.j2' %}
+
+
+## Description
+
+Stunnel is an open-source multi-platform application used to provide a universal
+TLS/SSL tunneling service.
+
+
+## How to use this Image
+
+The Stunnel Container Image launches `stunnel` by default using a minimal
+configuration file that only sets:
+- run in foreground
+- load further configuration files from `/etc/stunnel/conf.d`
+
+Custom configuration files should be placed into the directory
+`/etc/stunnel/conf.d`.
+
+The container entry point configures TLS/SSL automatically by setting the key
+and certificate to the values of the environment variables `STUNNEL_KEY` and
+`STUNNEL_CERT`. If one of the environment variables is unset, then the
+entrypoint defaults to `/etc/stunnel/stunnel.key` for `STUNNEL_KEY` and
+`/etc/stunnel/stunnel.pem` for `STUNNEL_CERT`.
+
+The entrypoint can setup a single service via environment variables so that the
+user doesn't have to write and mount their own configuration file. This can be
+setup via the environment variables `STUNNEL_SERVICE_NAME`, `STUNNEL_ACCEPT` and
+`STUNNEL_CONNECT`:
+
+- `STUNNEL_SERVICE_NAME`: name or otherwise unique identifier of the service
+  (used for documentation purpose only)
+
+- `STUNNEL_ACCEPT`: address on which new connections should be accepted, can be
+  either a hostname + port number or just a port number (then localhost is
+  assumed as the host)
```

Comment:
```suggestion
- `STUNNEL_ACCEPT`: address on which new connections should be accepted. It can be
  either a hostname and a port number or just a port number (in which case, localhost is
  assumed to be the host)
```